lib/takeaway.rb

```diff
@@ -0,0 +1,40 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+require 'menu'
+require 'order'
+Dotenv.load
+
+class Takeaway
+  attr_reader :order_klass, :order, :order_time
+
+  def initialize(order_klass)
+    @order_klass = order_klass
+    @order = nil
+  end
+
+  def new_order(menu_klass)
+    @order = order_klass.new(menu_klass)
+  end
+
+  def place_order(order, quantity, total, number)
+    fail "Wrong total!" if total != order.calculate_cost(order)
+    @order_time = Time.now + 3600
+    send_sms(number)
+  end
+
+  private
+
+  def send_sms(number)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `number`. If it's necessary, use `_` or `_number` as an argument name to indicate that it won't be used. You can also write as `send_sms(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)